### DISCUSSION TITLE
BIO

### DISCUSSION THREAD

**Discussion by Musabek03** [community/community#54160] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

-My name is Musabek
-I'm beginner PHP backend developer

**Comment by emilburganov** [comment]
Hello, I'm Emil, beginner Web Developer.

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @Musabek03 and @emilburganov, happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.
